network/interlink/reference-content/statuses.mdx

```diff
@@ -0,0 +1,39 @@
+| **Requesting**        | InterLink has been created successfully on the Scaleway side, provisioning must now be requested from the chosen partner. <br/><br/> You must contact your partner to request that they provision your InterLink. Either a contact email address, or a link to the partner’s portal is provided via the Scaleway API. Scaleway does not make the request to the partner, this is the responsibility of the user.<br/><br/> The InterLink will remain in `Requesting` status until the partner accepts the request, at which point it moves into `Provisioning` status. Alternatively, the partner may refuse the request, or the the request may time out. |
+| **Failed**          | Scaleway was unable to create the InterLink. <br/><br/> This usually occurs when there is a configuration error on the Scaleway side while attempting to create the InterLink. Our technical team is automatically made aware of such a problem, and will intervene to resolve it. When the problem is resolved, the InterLink should move to `Requesting` status. Wait for technical intervention, or open a support ticket if you require further help.
+| **Provisioning**      | The partner has accepted the InterLink request, and is now provisioning the connection at their end. <br/><br/> The InterLink is not yet operational: traffic cannot yet flow. You must wait for the partner to finish provisioning the connection. When provisioning is complete, the InterLink should move to `Active` status. |
+| **Expired**           | No response was received from the selected partner in the 10 days following the creation of the InterLink on the Scaleway side. <br/><br/> If Scaleway does not receive a response from the selected partner after 10 days, the InterLink moves to `Expired` status. It is not possible to take action to move this InterLink to a different status: you must delete the InterLink, and create a new order if you wish to try again. |
+| **Refused**           | The partner refused the request to provision the InterLink. <br/><br/> It is not possible to take action to move this InterLink to a different status: you must delete the InterLink, and create a new order if you wish to try again with a different partner or configuration.  |
+| **Active**            | The InterLink is in a ready state, and traffic can flow. <br/><br/> The InterLink has been provisioned on both the Scaleway side and the partner side, and is now in an active state. Traffic can flow across the InterLink (presuming that allowed routes lists, route propagation, and an attached VPC are correctly configured). |
+| **Limited Connectivity** | The InterLink is provisioned, but IP connectivity is limited. <br/><br/> This may be the case if either, but not both of, the BGP IPv4 or IPv6 sessions are down. |
```

Comment:
```suggestion
| **Limited Connectivity** | The InterLink is provisioned, but IP connectivity is limited. <br/><br/> This may be the case if either, but not both of, the BGP, IPv4, or IPv6 sessions are down. |
```